README.rst

```diff
@@ -123,20 +123,31 @@ Install `VSCode <https://code.visualstudio.com/download>`_
 
 .. code-block:: bash
 
-    # Setup a virtual env
+    # 1. Setup a virtual env
     virtualenv env
     . env/bin/activate
 
-    # Install pyls
+    # 2. Install pyls
     pip install .
 
-    # Install the vscode-client extension
+    # 3. Install the vscode-client extension
     cd vscode-client
     yarn install
 
-    # Run VSCode which is configured to use pyls
+    # 4. Run VSCode which is configured to use pyls
     # See the bottom of vscode-client/src/extension.ts for info
     yarn run vscode -- $PWD/../
+    
+    # How to view your change?
+    # Redo step 2 and 4 after making the change
+    
+    # How to add new plugins?
+    # Wrtie your code in pyls/plugins and corresponding test in test/plugins
+    # Resgiter your plugin in setup.py (under entry_points.plys)
+        
+    # Troubleshooting for palantirian
+    # Error: self signed certificate in certificate chain
+    # Try connect to RacerXGuestPortal (if not working, connect back to RacerX)
```

Comment:
I don't understand this part. What's `RacerXGuestPortal` and why is it needed?

Reply:
Hi, sorry if this is confusing. For Palantirians who work on this repo, this error shows up because we are by default connect to `RacerX` which is the internal WIFI. If this is too confusing for other developers, I can remove this part. I think the remaining would be valuable for everyone. Thanks

Reply:
I think if you rephrase a little bit, it'd be fine. Something like

```python
# For people who work in Palantir:
# If you encounter an error about a self signed certificate in certificate chain,
# Please try to connect to RacerXGuestPortal (if not working, connect back to RacerX)
```

This is just a suggestion. Please adjust it to what you consider is the right phrasing in this case.